### ISSUE TITLE
Preparing the data matrices - Splitting into train and validation sets

### ISSUE THREAD

**Issue by AbhimanyuSethi-98** [arya46/portfolio_notebooks#1]
Hi, I notice you've split the training set to make training and validation sets. However, don't we already have a validation set? Could you please explain your reasoning behind the same?

**Comment by arya46** [comment]
@AbhimanyuSethi-98 Yes, there is a separate validation set provided by the host. I have avoided using it because of computing resources limitations. If you don't have such an issue, you should definitely use the validation set, instead of splitting the training data.

**Comment by AbhimanyuSethi-98** [comment]
Wow hey thanks for the prompt response! I'm working on Colab itself sadly so I might have to follow the same procedure as you. Do you think I would run out of RAM if I took the validation set as-is (if that was the case with you).
Again, thanks for the response and the help :)

**Comment by arya46** [comment]
@AbhimanyuSethi-98 Colab provides GPU usage for limited hours only. If you wish to use the validation set, it will increase the training time. And you might have to suffer multiple timeouts before the model can converge. 
The timeout provided by Colab is generally 10-12 hours long. 

Using a validation set won't necessarily crash your system. You can reduce the batch_size and prevent such scenarios.

**Comment by AbhimanyuSethi-98** [comment]
@arya46 thanks man. Really appreciate it.
